edit-post/store/utils.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Given a selector returns a functions that returns the listener only
+ * if the returned value from the selector changes.
+ *
+ * @param  {function} selector Selector.
+ * @param  {function} listener Listener.
+ * @return {function}          Listener creator.
+ */
+export const onChangeListener = ( selector, listener ) => {
```

Comment:
I like this helper, can it be part of our data module instead of being part of the edit-post?

Reply:
Good call, isn't it the pattern that @aduth is sharing all over again on Slack? 😃

Reply:
Yes, my reluctance to introduce this pattern to the data module comes from it being a generic FP pattern and not something really specific to the data module. I understand the convenience though.